src/build.ts

```diff
@@ -157,9 +160,46 @@ export async function build(
     await effects.copyFile(sourcePath, outputPath);
   }
 
-  // Copy over the imported modules.
+  // Resolve some special additional downloads.
```

Comment:
I think this means we’ll need a config option that allows you to enumerate extra files to download from npm as part of build. (We don’t need them in preview because we can lazily fetch them when the code is evaluated.) But we can add that as a future enhancement.